src/App.tsx

```diff
@@ -1,35 +1,3 @@
-import { useState } from 'react'
```

Comment:
You should move all folders to src: components, pages, routes, assets
and please rename routes to router

Reply:
fixed